rca/programmes/models.py

```diff
@@ -792,9 +793,17 @@ class ProgrammePage(TapMixin, ContactFieldsMixin, BasePage):
     search_fields = BasePage.search_fields + [
         index.SearchField("programme_description_copy", boost=2),
         index.SearchField("pathway_blocks", boost=2),
-        index.SearchField("what_you_will_cover_blocks", boost=2),
+        index.SearchField(
+            "what_you_will_cover_blocks", boost=2
+        ),  # only works for 'accordion_block' type
```

Comment:
**Suggestion**

I think it would be clear to stat specifically _what_ only works for the accordion block here, just reading this I got a bit puzzled due to the indentation and placement.

Reply:
Good point! Thanks, I've updated the code in b4d80600